src/commands/aksOpenKubectlPanel/aksOpenKubectlPanel.ts

```diff
@@ -0,0 +1,74 @@
+import * as vscode from "vscode";
+import * as k8s from "vscode-kubernetes-tools-api";
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { KubectlDataProvider, KubectlPanel } from "../../panels/KubectlPanel";
+import { CommandResponse } from "../../plugins/shared/pluginResponses";
+import { getExtension } from "../utils/host";
+import { getKubectlCustomCommands } from "../utils/config";
+import { failed } from "../utils/errorable";
+import { createTempFile } from "../utils/tempfile";
+import { getReadySessionProvider } from "../../auth/azureAuth";
+import { selectClusterOptions, SelectClusterOptions } from "../../plugins/shared/clusterOptions/selectClusterOptions";
+import { checkExtension, handleExtensionDoesNotExist } from "../utils/ghCopilotHandlers";
+
+const GITHUBCOPILOT_FOR_AZURE_VSCODE_ID = "ms-azuretools.vscode-azure-github-copilot"
```

Comment:
💡 this is good for now, if I will get chance, I can do refactor under this PR. 

Otherwise - Lets do it in separate clean up PR. Thanks.

Reply:
Will refactor this in follow up PR